app/views/administrate/application/show.html.erb

```diff
@@ -43,9 +43,13 @@ as well as a link to its edit page.
 <section class="main-content__body">
   <dl>
     <% page.attributes.each do |title, attributes| %>
-      <fieldset class="<%= "field-unit--nested" if title.present? %>">
+      <section class="<%= "field-unit--nested" if title.present? %>">
```

Comment:
Semantically, `<fieldset>` elements typically wrap interactive form controls. Since the `show` templates present information without the opportunity to edit that information, this change improves the semantics of what these elements are communicating.

Having acknowledged that, rendering multiple `<h1>` elements [poses some accessibility risks](https://www.w3.org/WAI/tutorials/page-structure/headings/), and nesting a `<nav>` element (a role that comes with its own set of semantics and requirements) inside an `<h1>` poses additional risks.

Are there other structural changes we could make to achieve a similar outcome without the risks?

Reply:
What element (or elements) would you recommend to replace my `<h1>` and `<nav>` combination? Do you recommend something in particular? 🤔